### PR TITLE
Polishing XML files #185

### PR DESCRIPTION
**Pull request by kazuki43zoo**
Please review #185 .

### CODE REVIEW

File: projectName-web/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -17,6 +16,10 @@
         </param-value>
     </context-param>
 
+    <listener>
+        <listener-class>org.terasoluna.gfw.web.logging.HttpSessionEventLoggingListener</listener-class>
```
Comment (making) [inline projectName-web/src/main/webapp/WEB-INF/web.xml]: This line should be added by another commit.

Comment (kazuki43zoo) [inline projectName-web/src/main/webapp/WEB-INF/web.xml]: This line is not added.  It replaced definition order.

[Before]
* `ContextLoaderListener`
* `HttpSessionEventLoggingListener`
* `context-param` for `ContextLoaderListener`

[After]
* `ContextLoaderListener`
* `context-param` for `ContextLoaderListener`
* `HttpSessionEventLoggingListener`

Comment (making) [inline projectName-web/src/main/webapp/WEB-INF/web.xml]: sorry

Comment (btshimizukza) [inline projectName-web/src/main/webapp/WEB-INF/web.xml]: thanks!
